pp/go/cppbridge/prometheus_relabeler.go

```diff
@@ -390,14 +390,24 @@ type stdVector struct {
 	endOfStorage uintptr
 }
 
+type bareBonesVector struct {
+	memory   uintptr
+	capacity uint32
+	size     uint32
+}
+
+type roaringBitset [40]byte
+
 // InnerSeries - go wrapper for C-InnerSeries.
 //
 //	size - number of timeseries processed;
 //	data - pointer for vector with timeseries;
 type InnerSeries struct {
 	size uint64
 	//nolint:unused // for cpp-bridge, used in cpp
-	data stdVector
+	data bareBonesVector
+	//nolint:unused // for cpp-bridge, used in cpp
+	trackStaleNans roaringBitset
```

Comment:
I thought that inner series will be just uintptr with fastcgo methods. Do we really need maintain this structure?

Reply:
We decided to fix it in a separate PR